Voter/AclVoter.php

```diff
@@ -22,13 +22,39 @@
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
 
+if (class_exists(\Symfony\Component\Security\Core\Security::class)) {
+    /**
+     * @internal
+     */
+    trait AclVoterTrait
+    {
+        public function vote(TokenInterface $token, $subject, array $attributes)
+        {
+            return $this->doVote($token, $subject, $attributes);
+        }
+    }
+} else {
+    /**
+     * @internal
+     */
+    trait AclVoterTrait
+    {
+        public function vote(TokenInterface $token, mixed $subject, array $attributes): int
```

Comment:
the way to support multiple versions without such trait is actually to add the return type without adding the parameter type, thanks to variance rules.

however, note that adding the return type requires releasing this as a major version of the package, as this class is not final or internal.

Reply:
IMO, since this major version was previously tagged as it supports sf 7, we should fix it here, and in the next major remove all the unnecessary code and leave the voter clean again. wdyt?

Reply:
with major release or not,  we need a quick solution for this error, symfony 7.2 its coming and we cant use this bundle with sf7.